frontend/apps/filemanager/filemanager.lua

```diff
@@ -197,6 +197,19 @@ function FileManager:init()
                     end,
                 }
             },
+            {
+                {
+                    text = _("File information"),
+                    enabled = FileManager:fileInformation(file) and true or false,
```

Comment:
`fileInformation` is a very expensive operation because it might need to parse and load the whole book if user hasn't opened it before. If you set enabled right here, this function will be called even if the user does not click on the `File information` button. Every time we add a feature, we should make sure we don't compromise performance for other part of the system unless really necessary.

I suggest we make this button enabled by default and handle the error in callback function. Then we can also provide feedback to the user as why KOReader cannot fetch metadata for a selected entry using `InfoMessage` widget.

I also suggest renaming this button to `Book information` since that's what it actually does ;)